src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj

```diff
@@ -29,8 +29,10 @@
 
       Note: PrivateAssets="ContentFiles" ensures that projects referencing Microsoft.CodeAnalysis.Common package
       will import everything but content files from Microsoft.CodeAnalysis.Analyzers, specifically, analyzers.
+
+      Disabled in net8.0 until https://github.com/dotnet/roslyn-analyzers/issues/6059 is resolved.
     -->
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" Condition="'$(DotNetBuildFromSource)' != 'true'" PrivateAssets="ContentFiles" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" Condition="'$(DotNetBuildFromSource)' != 'true' AND '$(TargetFramework)' != 'net8.0'" PrivateAssets="ContentFiles" />
```

Comment:
PublicApi analyzer fails - it suggests changing `IntPtr` to `nint` for net8.0 - https://github.com/dotnet/roslyn-analyzers/issues/6059:

```diff
-static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromImage(System.IntPtr peImage, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
+static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(nint metadata, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
```